### PR TITLE
Add test case for issue #200

### PR DESCRIPTION
**Pull request by emestee**
GitHub-Issue: 200

### CODE REVIEW

File: php-mode.el [hunk 1]
```diff
@@ -11,7 +11,7 @@
 (defconst php-mode-version-number "1.15.3"
   "PHP Mode version number.")
 
-(defconst php-mode-modified "2015-01-31"
+(defconst php-mode-modified "2015-02-02"
```
Comment (syohex) [inline php-mode.el]: You modified only test, not `php-mode` itself. Is it necessary to change this value ?

Comment (emestee) [inline php-mode.el]: Perhaps I am misreading the readme. I will revert it if you prefer. The test is there to prove that #200 should be closed, that's it.

Comment (syohex) [inline php-mode.el]: @ejmr How about ?

**Comment by emestee** [comment]
I've reapplied the commit w/o the date change. I am wondering however if the test should be more elaborate (i.e. if there is actual highlighting and stripping)

**Comment by ejmr** [comment]
The more comprehensive the test, the better of course.  But thank you for providing this test since it's useful to have as it is now.

**Comment by syohex** [comment]
@emestee :+1:

**Pull request merged**